ios/RNPilgrimSdk.m

```diff
@@ -50,7 +50,7 @@ - (dispatch_queue_t)methodQueue
                  rejecter:(RCTPromiseRejectBlock)reject) {
     [[FSQPPilgrimManager sharedManager] getCurrentLocationWithCompletion:^(FSQPCurrentLocation * _Nullable currentLocation, NSError * _Nullable error) {
         if (error) {
-            reject(@"get_current_location", @"An error occurred getting your current location", error);
+            reject(@"get_current_location", error.localizedDescription, error);
```

Comment:
any reason not to change the `code` as well? i think `error.code` could be more helpful than `"get_current_location"`

Reply:
it's null, i think we would need to add a property to internal error object but would likely require change to sdk not wrapper
```
(lldb) po error.code
<nil>
```